src/Canvas/Pixel.elm

```diff
@@ -0,0 +1,227 @@
+module Canvas.Pixel
+    exposing
+        ( put
+        , putMany
+        , line
+        , rectangle
+        , bezier
+        )
+
+import Canvas exposing (Canvas, Size, Point, DrawOp(..))
+import Color exposing (Color)
+import Array exposing (Array)
+
+
+put : Color -> Point -> DrawOp
```

Comment:
Re performance, I was curious about this, and I did a test that compared the speeds of..

0 Drawing a diagonal line by making a new imageData, x by x in its dimensions, putting in the pixels into that imageData, and then using drawImage() to draw it onto the canvas.

1 Drawing a diagonal line by drawing each pixel one at a time in this way.

I was surprised to find that method 1 worked faster, especially for longer length lines. I believe that as the pixels per region goes up, 0 becomes more efficient. So, in a horizontal or vertical line, there are as many pixels to draw as there are pixels in the region, so I would expect method 0 to be faster. That also means that `Pixel.rectangle` could be optimized by relying on something other than `put`.

Im down with eliminating `Point`, but if we do, how about we do `Int -> Int-> ..` instead? I dont know. Ill think about it more. I asked the elm slack general because I was curious about this question generally.